metaconfig-core/shared/src/main/scala/metaconfig/ConfDecoder.scala

```diff
@@ -109,14 +109,14 @@ object ConfDecoder {
       implicit ev: ConfDecoder[A],
       classTag: ClassTag[A]
   ): ConfDecoder[Map[String, A]] =
-    CanBuildFromDecoder.map[A]
+    FactoryDecoder.map[A]
 
   implicit def canBuildFromConfDecoder[C[_], A](
```

Comment:
I’m suspect this is a binary breaking change. I think it would be better to limit this change to 2.13+

Reply:
I just ran `coreJVM/mimaReportBinaryIssues` and got the following errors

```
[error]  * object metaconfig.internal.CanBuildFromDecoder does not have a correspondent in current version
[error]    filter with: ProblemFilters.exclude[MissingClassProblem]("metaconfig.internal.CanBuildFromDecoder$")
[error]  * class metaconfig.internal.CanBuildFromDecoder does not have a correspondent in current version
[error]    filter with: ProblemFilters.exclude[MissingClassProblem]("metaconfig.internal.CanBuildFromDecoder")
```

I'll try to work around it.